assets/js/classes/CharacterFeatures.js

```diff
@@ -0,0 +1,76 @@
+class CharacterFeatures{
+    character = document.querySelector('#character');
+    parentElementHeight = this.character.parentNode.offsetHeight;
+    halfHeightOfParentElement = this.parentElementHeight * (50/100);
```

Comment:
O que acha de deixar assim, fica mais explicito que vc quer 50% do valor
```suggestion
    halfHeightOfParentElement = this.parentElementHeight * 0.5;
```